endpoints.json

```diff
@@ -25,5 +25,31 @@
         }
       ]
     }
+  },
+
+  "GET /api/articles/:article_id": {
+    "description": "serves a single article by its id",
+    "exampleResponse": {
+      "article": {
+        "author": "jessjelly",
+        "title": "Running a Node App",
+        "article_id": 1,
+        "body": "This is part two of a series on how to get up and running with Systemd and Node.js. This part dives deeper into how to successfully run your app with systemd long-term, and how to set it up in a production environment.",
+        "topic": "coding",
+        "created_at": "2020-11-07T06:03:00.000Z",
+        "votes": 0,
+        "article_img_url": "https://images.pexels.com/photos/11035380/pexels-photo-11035380.jpeg?w=700&h=700"
+      }
+    },
+    "possibleErrors": [
+      {
+        "status": 404,
+        "description": "Article not found"
+      },
+      {
+        "status": 500,
+        "description": "Internal server error"
+      }
+    ]
```

Comment:
This is an interesting idea, but not something a user should need to know. We should expect the user to only see the valid ways to use this api in this json file so it should not contain any errors here.